bedrock/firefox/templates/firefox/os/faq.html

```diff
@@ -21,160 +21,206 @@ <h1 class="large">{{_('Firefox OS FAQ')}}</h1>
+                            {{_('While every other phone today is static, giving you the exact same apps each time you turn it on, Firefox OS is always changing to offer exactly the apps you need at any moment in your life.')}}
+                            {{_('It adapts so you can live every moment to its fullest.')}}
+                        </p>
+                        <p>
+                            {{_('Firefox OS is brought to you by Mozilla, a global nonprofit organization that emphasizes principle over profit.')}}
+                            {{_('We believe that the Web is a shared public resource to be cared for, not a commodity to be sold.')}}
+                            {{_('We work with a worldwide community to create open source products like Mozilla Firefox and to innovate for the benefit of the individual and the betterment of the Web.')}}
+                            {{_('The result is great products built by passionate people and better choices for everyone.')}}
+                        </p>
+                    {% endif %}
+                    </dd>
+                </dl>
+            </div>
+        </section>
+
+        <section>
+            <h2 data-accordion-role="tab">{{_('Buying Firefox OS')}}</h2>
+            <div data-accordion-role="tabpanel">
+                <dl class="faq">
+                    <dt>{{_('What phones will Firefox OS be available on?')}}</dt>
+                    <dd>
+                    {% if l10n_has_tag('fxos_faq_updates') %}
+                        {{_('Since launching, Firefox OS smartphones are available in over 40 markets from multiple global operators on a number of different devices, across different price device categories. These include handsets from Alcatel ONETOUCH, ZTE, Huawei and LG.')}}
+                    {% else %} 
+                        {% trans link=url('firefox.os.devices') %}
+                        To see a full list of available Firefox OS phones, please visit the <a href="{{ link }}">devices page</a>.
+                        {% endtrans %}
+                    {% endif %}
+                    </dd>
+
+                    <dt>{{_('Where can I buy Firefox OS?')}}</dt>
+                    <dd>
+                    {% if l10n_has_tag('fxos_faq_updates') %}
+                        {% trans link=url('firefox.os.devices') %}
+                        The full list of where to buy Firefox OS devices can be found at the <a href="{{ link }}">devices page</a>.
+                        {% endtrans %}
+                    {% else %} 
+                        {% trans link=url('firefox.os.devices') %}
+                        To purchase a Firefox OS phone, please visit the <a href="{{ link }}">devices page</a> and select your available country.
+                        {% endtrans %}
+                    {% endif %}
+                    </dd>
+
+                    <dt>{{_('What will be the price of the phone?')}}</dt>
```

Comment:
This line is currently live and is not part of the copy updates. Will leave as is.